oap-server/server-query-plugin/debugging-query-plugin/src/main/java/org/apache/skywalking/oap/query/debug/DebuggingHTTPHandler.java

```diff
@@ -176,10 +176,11 @@ public String queryBasicTraces(@Param("service") String service,
                                    @Param("pageNum") int pageNum,
                                    @Param("pageSize") int pageSize
     ) {
-        String serviceId = IDManager.ServiceID.buildId(service, Layer.nameOf(serviceLayer).isNormal());
+        Optional<String> serviceId = service.map(
```

Comment:
Why change to `Optional<String> serviceLayer` and `serviceLayer.orElseThrow()` if it’s required?